### DISCUSSION TITLE
Question about versioning

### DISCUSSION THREAD

**Discussion by ADD-ACS** [jstedfast/MailKit#1782] [Q&A] [answered]
I uppgraded MailKit NuGet package from version `4.7.0` to `4.7.1.1` and I realised dlls are almost the same, in fact they have the same exact weight. Inspecting with dnSpy I see `AssemblyInformationalVersion` and `AssemblyFileVersion` properties are the only ones that were updated, but not the same for `AssemblyVersion`, which has the value `4.7.0.0` in both cases. I expected the second dll to have `AssemblyVersion` with value `4.7.1.1`. Why does not have it?

Thanks in advance.

**Comment by jstedfast** [comment, marked as answer]
Because the API/ABI did not change.
